[compat] test_user_management.py.TestUserManagementRollingUpgrade.test_user_management...: User management rolling upgrade

## Summary
User management rolling upgrade

## Environment
- Branch: main `345cf9df9923a4d214bfe902b2bd2e723b24bfa3`
- Compat chain: stable-26-1-1 `247d5be0d7b2fbdeb3f8ca920edc23cf35ffcc36` -> stable-26-2-1 `abac06dc12ea0a91479fb3aabbf7e467cb102f82` -> stable-26-3-1 `b1e96c302f42b329d2387d4b4ae451347aa9c47d`
- CI: [29327138619](https://github.com/ydb-platform/ydb/actions/runs/29327138619)
- Reports: [try1](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_1/ya-test.html), [try2](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_2/ya-test.html), [try3](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_3/ya-test.html)

- **Full log:** https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_3/artifacts/logs/ydb/tests/compatibility/common/test-results/py3test/testing_out_stuff/test_user_management.py.TestUserManagementRollingUpgrade.test_user_management.rolling_stable-26-2-1_to_stable-26-3-1.log
- Failed attempts: try_1, try_2, try_3

## Reproduction
```
-DYDB_COMPAT_INIT_REF=stable-26-2-1 -DYDB_COMPAT_INTER_REF=stable-26-3-1 -DYDB_COMPAT_TARGET_REF=prestable-26-4
```

## Tests (1)
```
ydb/tests/compatibility/common/test_user_management.py.TestUserManagementRollingUpgrade.test_user_management[rolling_stable-26-2-1_to_stable-26-3-1]
```

## Error
```
ydb/tests/compatibility/common/test_user_management.py:53: in test_user_management
    session_pool.execute_with_retries(query)
contrib/python/ydb/py3/ydb/query/pool.py:259: in execute_with_retries
    return retry_operation_sync(wrapped_callee, retry_settings)
contrib/python/ydb/py3/ydb/retries.py:174: in retry_operation_sync
    for next_opt in retry_operation_impl(traced_callee, retry_settings, *args, **kwargs):
contrib/python/ydb/py3/ydb/retries.py:123: in retry_operation_impl
    result = YdbRetryOperationFinalResult(callee(*args, **kwargs))
contrib/python/ydb/py3/ydb/retries.py:171: in traced_callee
    return callee(*a, **kw)
contrib/python/ydb/py3/ydb/query/pool.py:257: in wrapped_callee
    return [result_set for result_set in it]
contrib/python/ydb/py3/ydb/_utilities.py:189: in __next__
    return self._next()
contrib/python/ydb/py3/ydb/query/base.py:87: in _next
    return super()._next()
contrib/python/ydb/py3/ydb/_utilities.py:180: in _next
    res = self.wrapper(next(self.it))
contrib/python/ydb/py3/ydb/query/session.py:513: in <lambda>
    lambda resp: base.wrap_execute_query_response(
contrib/python/ydb/py3/ydb/query/base.py:219: in decorator
    return func(rpc_state, response_pb, session, *args, **kwargs)
contrib/python/ydb/py3/ydb/query/base.py:236: in wrap_execute_query_response
    issues._process_response(response_pb)
contrib/python/ydb/py3/ydb/issues.py:264: in _process_response
    raise exc_class(_format_response(response_proto), response_proto.issues)
E   ydb.issues.PreconditionFailed: message: "Executing ESchemeOpAlterLogin" issue_code: 2029 severity: 1 issues { message: "Plain password is no longer supported, use hashed password instead" issue_code: 2029 severity: 1 } (server_code: 400120)```

--- a/ydb/core/tx/schemeshard/schemeshard__operation_alter_login.cpp
+++ b/ydb/core/tx/schemeshard/schemeshard__operation_alter_login.cpp
@@ -37,11 +37,6 @@ public:
                 case NKikimrSchemeOp::TAlterLogin::kCreateUser: {
                     const auto& createUser = alterLogin.GetCreateUser();
 
-                    if (!createUser.HasHashedPassword() || createUser.HasPassword()) {
-                        result->SetStatus(NKikimrScheme::StatusPreconditionFailed, "Plain password is no longer supported, use hashed password instead");
-                        break;
-                    }
-
                     NLogin::TLoginProvider::TCreateUserRequest request;
                     request.User = createUser.GetUser();
                     request.HashedPassword = createUser.GetHashedPassword();
@@ -76,11 +71,6 @@ public:
                 }
                 case NKikimrSchemeOp::TAlterLogin::kModifyUser: {
                     const auto& modifyUser = alterLogin.GetModifyUser();
-
-                    if (modifyUser.HasPassword()) {
-                        result->SetStatus(NKikimrScheme::StatusPreconditionFailed, "Plain password is no longer supported, use hashed password instead");
-                        break;
-                    }
 
                     NLogin::TLoginProvider::TModifyUserRequest request;
 

